### ISSUE TITLE
Problem when plotting with background field

### ISSUE THREAD

**Issue by knutfrode** [OpenDrift/opendrift#47]
Sometimes reader_x, reader_y contains NaN values, and basemodel.get_map_background() fails.
Could be fixed by something like:
```
reader_x[np.isnan(reader_x)] = <somevalue>
reader_y[np.isnan(reader_y)] = <somevalue>
```

**Comment by dharanidharan12** [comment]
In **OpenOil3D**
while running the sample Getting  **Please provide Reader object** Error
What I did Wrong???
Please Help!!!

**Comment by dharanidharan12** [comment]
Using **OpenOil3D**
Getting Error:
**OilProps' object has no attribute 'oil_water_surface_tension'**
What I did wrong???
Please Help!!!

**Comment by dharanidharan12** [comment]
**ShipDrift**
I getting error like  **'Reader' object is not iterable**
while using  o.run(steps=24 )
how to correct this issue.
please Help!!!

**Comment by knutfrode** [comment]
Solved by 5fa7164ef4c2ccf3bea43da0052941410923828c
